lib/next_ls.ex

```diff
@@ -171,6 +173,62 @@ defmodule NextLS do
     {:reply, symbols, lsp}
   end
 
+  # TODO handle `context: %{includeDeclaration: true}` to include the current symbol definition among
+  # the results.
+  def handle_request(%TextDocumentReferences{params: %{position: position, text_document: %{uri: uri}}}, lsp) do
+    file = URI.parse(uri).path
+    line = position.line + 1
+    col = position.character + 1
+
+    locations =
+      dispatch(lsp.assigns.registry, :databases, fn databases ->
+        for {database, _} <- databases do
```

Comment:
If you are going to flatten it below, you might as well use Enum.flat_map here instead of a comprehension

Reply:
You are absolutely right! Much better to do it in a single operation since it doesn't hurt readability either.

I've just pushed a commit that replaces the for comprehension with `Enum.flat_map` as you suggested.